Tests failing on fresh install

```
$ npm test

> metalsmith-html-tidy@1.1.1 test /Users/***/repos/metalsmith-html-tidy
> mocha test/

module.js:339
    throw err;
    ^

Error: Cannot find module 'debug'
    at Function.Module._resolveFilename (module.js:337:15)
    at Function.Module._load (module.js:287:25)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/thiago/repos/metalsmith-html-tidy/lib/index.js:3:13)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at Object.<anonymous> (/Users/thiago/repos/metalsmith-html-tidy/test/test.js:6:14)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Module.require (module.js:366:17)
    at require (module.js:385:17)
    at /Users/thiago/repos/metalsmith-html-tidy/node_modules/mocha/lib/mocha.js:220:27
    at Array.forEach (native)
    at Mocha.loadFiles (/Users/thiago/repos/metalsmith-html-tidy/node_modules/mocha/lib/mocha.js:217:14)
    at Mocha.run (/Users/thiago/repos/metalsmith-html-tidy/node_modules/mocha/lib/mocha.js:469:10)
    at Object.<anonymous> (/Users/thiago/repos/metalsmith-html-tidy/node_modules/mocha/bin/_mocha:404:18)
    at Module._compile (module.js:435:26)
    at Object.Module._extensions..js (module.js:442:10)
    at Module.load (module.js:356:32)
    at Function.Module._load (module.js:311:12)
    at Function.Module.runMain (module.js:467:10)
    at startup (node.js:136:18)
    at node.js:963:3
npm ERR! Test failed.  See above for more details.
```

--- a/package.json
+++ b/package.json
@@ -21,12 +21,13 @@
   "devDependencies": {
     "assert": "^1.3.0",
     "assert-dir-equal": "^1.0.1",
+    "debug": "^2.2.0",
     "eslint": "^2.9.0",
     "eslint-config-airbnb": "^9.0.1",
     "eslint-plugin-import": "^1.8.0",
     "eslint-plugin-jsx-a11y": "^1.2.0",
-    "eslint-plugin-react": "^5.1.1",
     "eslint-plugin-promise": "^1.0.8",
+    "eslint-plugin-react": "^5.1.1",
     "metalsmith": "^2.1.0",
     "metalsmith-debug": "^1.0.0",
     "mocha": "^2.4.5"

